tap_github/repository_streams.py

```diff
@@ -1907,6 +1907,457 @@ def query(self) -> str:
+                        resource_path: resourcePath
+                        color
+                        default: isDefault
+                      }
+                    }
+                  }
+                  locked
+                  active_lock_reason: activeLockReason
+                  closed
+                  is_answered: isAnswered
+                  answer {
+                    id: databaseId
+                    node_id: id
+                    body
+                    author {
+                    ... on User{
+                      node_id: id
+                      id: databaseId
+                      login
+                      avatar_url: avatarUrl
+                      html_url: url
+                      type: __typename
+                      site_admin: isSiteAdmin
+                      }
+                    }
+                    author_association: authorAssociation
+                  }
+                  answer_chosen_by: answerChosenBy {
+                  ... on User{
+                    node_id: id
+                    id: databaseId
+                    login
+                    avatar_url: avatarUrl
+                    html_url: url
+                    type: __typename
+                    site_admin: isSiteAdmin
+                    }
+                  }
+                  upvote_count: upvoteCount
+                  reactions(first: 100) {
+                    edges {
+                      node {
+                        reaction_type: content
+                        reacted_at: createdAt
+                        user {
+                          login
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+            rateLimit {
+              cost
+            }
+          }
+        """  # noqa: E501
+
+    category_object = th.ObjectType(
+        th.Property("node_id", th.StringType),
+        th.Property("slug", th.StringType),
+        th.Property("name", th.StringType),
+        th.Property("description", th.StringType),
+    )
+
+    answer_object = th.ObjectType(
+        th.Property("id", th.IntegerType),
+        th.Property("node_id", th.StringType),
+        th.Property("body", th.StringType),
+        th.Property("author", user_object),
+        th.Property("author_association", th.StringType),
+    )
+
+    labels_array = th.ArrayType(
+        th.ObjectType(
+            th.Property("id", th.IntegerType),
+            th.Property("node_id", th.StringType),
+            th.Property("created_at", th.DateTimeType),
+            th.Property("name", th.StringType),
+            th.Property("description", th.StringType),
+            th.Property("url", th.StringType),
+            th.Property("resource_path", th.StringType),
+            th.Property("color", th.StringType),
+            th.Property("default", th.BooleanType),
+        )
+    )
+
+    reactions_array = th.ArrayType(
```

Comment:
Since we will use this one twice (here and discussion comments), we could refactor it out to the `schema_objects.py` file.  Just would need to name it something distinct from the existing `reactions_object` there.